### PR TITLE
Divide commands and functions

### PR DESCRIPTION
**Pull request by brrscc**
Moved all of the command definitions, listeners, and input/context validations to commander.py

Kept all of the backend/data operations in recruit.py

### CODE REVIEW

File: commander.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import asyncio
+
+from discord import app_commands
+from discord.app_commands import commands
+
+import config
+from recruit import recruit
+
+
+class commander(commands.Cog):
```
Comment (Darcness) [inline commander.py:13]: https://peps.python.org/pep-0008/#class-names
Please and thank you.

File: commander.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+    # Command to define a user's nation & template
+    @commands.hybrid_command(name="register", with_app_command=True,
```
Comment (Darcness) [inline commander.py:16]: What line length is being enforced here? This seems kind of short.

File: recruit.py
Comment (Darcness) [inline recruit.py:1]: If you're going to perform this level of separation, then recruit shouldn't be a cog.  'Commander' is your cog and entry point, and the Discord objects that are necessary are fed to Recruit.

Also... how does any of this work if you're still loading the 'recruit' cog in main.py?

**Review by Darcness — CHANGES_REQUESTED** [review]

File: recruit.py
Comment (Darcness) [reply recruit.py:1]: I think most of this type of restructure has been done by UPC already.  Is there anything in this PR we want to keep?

**Review by nsupc — COMMENTED** [review]
@brrscc are we good to close this, or is any of the functionality you added missing from main?

**Pull request closed without merging**